### PR TITLE
statistics: more friendly output

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

After #3591, we use []float64 to represent hot peer loads, but output is no friendly.

image

After this pr:

image

### Check List

Tests

- Unit test

### Release note

```release-note
 None.
```

### CODE REVIEW

File: server/statistics/hot_regions_stat.go [hunk 1]
```diff
@@ -13,9 +13,25 @@
...
+	Stats          []HotPeerStatShow `json:"statistics"`
+}
+
+type HotPeerStatShow struct {
+	StoreID        uint64    `json:"store_id"`
+	RegionID       uint64    `json:"region_id"`
+	HotDegree      int       `json:"hot_degree"`
+	ByteRate       float64   `json:"flow_bytes"`
+	KeyRate        float64   `json:"flow_keys"`
+	AntiCount      int       `json:"anti_count"`
```
Comment (rleungx) [inline server/statistics/hot_regions_stat.go:35]: I don't think users can understand it.

File: server/api/trend.go [hunk 2]
```diff
@@ -152,9 +152,9 @@ func (h *trendHandler) getStoreFlow(stats statistics.StoreHotPeersStat, regionSt
 		return
 	}
 	if stat, ok := stats[storeID]; ok {
-		storeFlow = stat.TotalLoads[regionStatKind]
+		storeFlow = stat.TotalBytesRate
```
Comment (rleungx) [inline server/api/trend.go:155]: The variable name doesn't match its meaning.

File: server/api/trend.go [hunk 3]
```diff
@@ -152,9 +152,9 @@ func (h *trendHandler) getStoreFlow(stats statistics.StoreHotPeersStat, regionSt
...
 		for _, flow := range stat.Stats {
-			regionFlows = append(regionFlows, flow.GetLoad(regionStatKind))
+			regionFlows = append(regionFlows, flow.ByteRate)
```
Comment (rleungx) [inline server/api/trend.go:157]: ditto

File: server/statistics/hot_regions_stat.go [hunk 4]
```diff
@@ -13,9 +13,25 @@
...
+import (
+	"time"
+)
+
 // HotPeersStat records all hot regions statistics
 type HotPeersStat struct {
-	TotalLoads []float64     `json:"total_loads"`
-	Count      int           `json:"regions_count"`
-	Stats      []HotPeerStat `json:"statistics"`
+	TotalLoads     []float64         `json:"-"`
```
Comment (rleungx) [inline server/statistics/hot_regions_stat.go:22]: What does it used for?

**Comment by nolouch** [comment]
Does this affect the [`TIDB_HOT_REGION`](https://docs.pingcap.com/tidb/stable/information-schema-tidb-hot-regions#tidb_hot_regions) table?

File: server/statistics/hot_regions_stat.go (hunk 4 above)
Comment (lhy1024) [reply server/statistics/hot_regions_stat.go:22]: update metircs, such as https://github.com/tikv/pd/blob/ab8b5b39930bde93c1258fbcfbc5d8b00eb64e40/server/cluster/coordinator.go#L514

File: server/api/trend.go (hunk 2 above)
Comment (lhy1024) [reply server/api/trend.go:155]: only revert it https://github.com/tikv/pd/pull/3591/files#diff-e7c11b02e9457f3fd65407917fc0c632537ea493dd5499aa18dfc8bc2c067d52L156

Comment (rleungx) [reply server/api/trend.go:155]: So we need to make it better.

**Comment by lhy1024** [comment]
> Does this affect the [`TIDB_HOT_REGION`](https://docs.pingcap.com/tidb/stable/information-schema-tidb-hot-regions#tidb_hot_regions) table?

This pr will let  TIDB_HOT_REGION table `flow_byte` column work well.

But I found `region_count` column, and I don't know what it means.

image

File: server/statistics/hot_regions_stat.go (hunk 1 above)
Comment (Yisaer) [reply server/statistics/hot_regions_stat.go:36]: But I think it's helpful for us to debug.

**Review by Yisaer — APPROVED** [review]

File: server/api/trend.go (hunk 2 above)
Comment (lhy1024) [reply server/api/trend.go]: done

File: server/api/trend.go (hunk 3 above)
Comment (lhy1024) [reply server/api/trend.go]: done

**Review by rleungx — APPROVED** [review]

**Pull request merged**